### ISSUE TITLE
Copyright starts with `Just`

### ISSUE THREAD

**Issue by nomeata** [ddssff/cabal-debian#41]
```
/tmp $ cabal unpack quickcheck-unicode
cdUnpacking to quickcheck-unicode-1.0.0.1/
/tmp $ cd quickcheck-unicode-1.0.0.1/
/tmp/quickcheck-unicode-1.0.0.1 $ ~/build/haskell/cabal-debian/.cabal-sandbox/bin/cabal-debian  --official 
/tmp/quickcheck-unicode-1.0.0.1 $ head -n 1 debian/copyright 
Just Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

**Comment by ddssff** [comment]
Darn, thought I had fixed that.

**Comment by ddssff** [comment]
Fixed in version 4.27.1.

**Comment by ddssff** [comment]
Never sure whose responsibility it is to close issues on github.  In bugzilla you went to resolved/fixed and then QA would move it to closed.

**Comment by nomeata** [comment]
It’s your responsibility as the maintainer, I believe. Note that you can actually close them directly with the ticket, by including `fixes #41` in the commit log.
